### ISSUE TITLE
ClickUp Trigger - Create New Credentials error

### ISSUE THREAD

**Issue by JustMaier** [n8n-io/n8n#1025]
**Describe the bug**
When trying to add credentials to the ClickUp Trigger node, I received the following message:
```
Credential type not known
Credentials of type "clickupApi" are not known.
```

**To Reproduce**
1. Add a ClickUp Trigger Node and open it
2. Click on "-Create New-" in the clickupApi dropdown
3. See error

[Video Example](https://www.loom.com/share/4c3ed9f988984db0b695702f8258b28b)

**Expected behavior**
I should be taken to the New Credentials form to add my API key.

**Environment (please complete the following information):**
 - OS: Windows 10 2004
 - n8n Version: 0.86.1
 - Node.js Version: 12.10.0

**Additional context**
Ran locally using `npx n8n`

**Comment by RicardoE105** [comment]
@JustMaier thanks for letting us know.  Will check this out and let you know.

**Comment by RicardoE105** [comment]
@JustMaier got fixed, we will let you know when is released. Again, thanks very much.

**Comment by janober** [comment]
Got merged. I will update here once it got released.

**Comment by JustMaier** [comment]
In my 10 years on GitHub, I've never seen a team move so fast on an issue. You guys are amazing!
Looking forward to getting involved with this awesome project :)

**Comment by janober** [comment]
Really amazing to hear @JustMaier ! Thanks a lot!
That is honestly a field I still think we have to improve generally. We are sometimes super fast (like in your case) but in other cases still way to slow because of team-size, missing processes, and the sheer amount of work. But we aim to deliver everybody the same amazing experience you had asap!

Very happy to have you on board! Have fun with n8n!

**Comment by janober** [comment]
ClickUp Trigger node got released with n8n@0.87.0
